### ISSUE TITLE
Server mod rejections Loot tweaker, Recurrent Complex

### ISSUE THREAD

**Issue by Syneticax** [xJon/Hexxit-II#240]
### Was this help requested before?

- [X] No, it was not.

### If you have any, please provide a list of additional mods

Optifine

### Please describe what you need help with

Hi xJon,
I have a problem installing on a server (Exaroton). I have included the entire server file (unpacked) as a whole folder under Mods. (It's been a long time since I created servers like this) 
The server also starts completely. But unfortunately none of us can enter the server. 

Report:
Server mod rejections: 
LootTweaker: Failed mod´s custom NetworkCheckHandler FMLMod:loottweaker(0.3.1)
Recurrent Complex: Failed mod´s custom NetworkCheckHandler FMLMod:reccomplex(1.4.8.5)

No issue for Singleplayer

I couldn't find this problem here. (Forum Noob) So if this problem has already been addressed, I apologize very much.

What did i do wrong? What do I have to consider when I install the modpack on the server? Apparently I did something wrong...
![Screenshot 2024-05-22 185046](https://github.com/xJon/Hexxit-II/assets/170558308/3950e1cc-63af-40b5-abe0-403ab55a294d)
![Screenshot 2024-05-22 192132](https://github.com/xJon/Hexxit-II/assets/170558308/b04a308a-b35e-4e4a-a0c0-685e6f2da065)
![Screenshot 2024-05-22 192144](https://github.com/xJon/Hexxit-II/assets/170558308/f46b1ed8-bf88-44c8-9faa-713a02130a26)
![Screenshot 2024-05-22 192202](https://github.com/xJon/Hexxit-II/assets/170558308/a806ddb1-6b7e-4b94-8d63-ebf33aef0926)

Thank u so much
We long for the old Hexxit nostalgia
ANd thanks for updating this mod :)

**Comment by xJon** [comment]
> I have included the entire server file (unpacked) as a whole folder under Mods.

You need to include the entire server file, unpacked, as the main folder (not under `Mods`).

**Comment by Syneticax** [comment]
I did as you said. 
![Screenshot 2024-05-22 193414](https://github.com/xJon/Hexxit-II/assets/170558308/e9867494-6f76-423a-9b4c-f7f584922182)

Same issue.

**Comment by xJon** [comment]
It needs to be unzipped, and then the forge jar needs to be launched (instead of the minecraft jar).

**Comment by Syneticax** [comment]
By the way, I'm sorry about the bad English. I'm from Germany

**Comment by xJon** [comment]
> What do you mean by launch forge instead of minecraft? We launch the game with the technic launcher..

I meant with your server host - you need to make sure you launch the right jar file. I don't have experience with exaroton so you might need to ask them how to change the jar file that is launched. If you want to consider alternative hosts, Hexxit II's official server hosting is https://www.akliz.net/hexxit which has built-in support for the pack (including updating to future versions).

**Comment by Syneticax** [comment]
I'll get in touch with them. Can you tell me whether curseforge Hexxit 2 is legit? could I also try the whole thing via curseforge?

**Comment by xJon** [comment]
No - the only place to get the Hexxit II client is via the Technic distribution (which could be accessed via the Technic Launcher or other launchers that support it, e.g. Prism Launcher).

**Comment by xJon** [comment]
Yes, that's right!

**Comment by Syneticax** [comment]
yeah... the host has a technic launcher support... well hidden.. just changed to that... works...
maybe im going to cry now :)
Thank you for everything <3
